internal/meter/discovergy.go

```diff
@@ -75,7 +76,7 @@ func NewDiscovergyFromConfig(other map[string]interface{}) (api.Meter, error) {
 	}
 
 	uri := fmt.Sprintf("%s/last_reading?meterId=%s", discovergyAPI, meterID)
-	power, err := provider.NewHTTP(log, http.MethodGet, uri, headers, "", false, ".values.power", 0.001)
+	power, err := provider.NewHTTP(log, http.MethodGet, uri, headers, "", false, ".values.power", 0.001, 10 * time.Second)
```

Comment:
Ich habe hier auch `request.Timeout` probiert. Aber das hat nicht geklappt und ich habe in dem Build auch keine Aussagekräftige Fehlermeldung gesehen.

Reply:
Nimmst Du VSCode? request importiert?

Ich würde aber das Setzen des Timeouts in eine methode packen, dann musst DU es hier einfach gar nicht machen da die defaults ja funktionieren!

Reply:
Danke, jetzt habe ich es verstanden.

Ich hatte eine veraltete go version und hatte keine Lust zu aktualisieren und dachte für die Kleinigkeit reicht der docker build. Jetzt habe ich doch mein go aktualisiert.